From 2c7c37684c98ecef57742238a98a2968b7738d3e Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Sun, 3 Jul 2016 17:24:03 +0300
Subject: [PATCH 0336/1345] dts: eip197: Add additional eip197 interrupts

commit  576466a7c56c9f904859237b94e1b02a2d86c29e from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

the eip197 exposes several interrupts which are used
by the DDK driver, this commit adds all 4 rings and global
interrupts support

Change-Id: I8c0f74616d7be67592be3bdd5ac2db68176284b0
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30917
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 7e139ee..28f1c19 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -363,7 +363,12 @@ eip197: eip197@800000 {
 	compatible = "marvell,eip197";
 	reg = <0x800000 0x200000>;
 	dma-coherent;
-	interrupts = <GIC_SPI 54 IRQ_TYPE_LEVEL_HIGH>;
+	interrupts = <GIC_SPI 54 IRQ_TYPE_LEVEL_HIGH>,
+			<GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>,
+			<GIC_SPI 56 IRQ_TYPE_LEVEL_HIGH>,
+			<GIC_SPI 57 IRQ_TYPE_LEVEL_HIGH>,
+			<GIC_SPI 58 IRQ_TYPE_LEVEL_HIGH>;
+	interrupt-names = "ring0", "ring1", "ring2", "ring3", "global";
 	clocks = <&gateclk 26>;
 	status = "disabled";
 };
-- 
1.7.9.5

